src/components/DpTabs/DpTabs.stories.mdx

```diff
@@ -0,0 +1,63 @@
+import { Meta } from '@storybook/addon-docs'
+import DpTabs from './DpTabs'
+
+<Meta
+  title="Components/Tabs"
+  component={DpTabs} />
+
+# DpTabs
+
+DpTabs is a wrapper for DpTab component through `<slot />`.
+
+DpTabs component is a simple user interface (tabs UI) for organizing related content and occupying a compact space.
+
+The DpTab components are aligned horizontally, and each tab is associated with its header.
+
+Components DpTabs and DpTab contain `<slot />`.
+
+## Default usage
+
+```html
+<dp-tabs
+    tab-size="medium"
+    :active-id="activeTabId"
+    use-url-fragment
+    @change="setActiveTabId"> // call a method
```

Comment:
Please use html comments here.

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/344/commits/579d5acd26d7468522a98a237c8a945dc6411efd